lib/lotus/routing/parsing/json_parser.rb

```diff
@@ -10,6 +17,8 @@ def mime_types
 
         def parse(body)
```

Comment:
We should add API doc to document this new error here and for `Parser`.
Please remember to update README section as well. Thank you!